### PR TITLE
amp-bind: Expose encodeURI and encodeURIComponent in bind

### PR DESCRIPTION
**Pull request by kmh287**
Adds support for `encodeURI` and `encodeURIComponent` in bind. 

Partial for #8700 

/to @choumx

### CODE REVIEW

File: extensions/amp-bind/0.1/test/test-bind-expression.js [hunk 1]
```diff
@@ -268,6 +268,11 @@ describe('BindExpression', () => {
     }).to.throw(unsupportedFunctionError);
   });
 
+  it('should support encodeURI and encodeURIComponent', () => {
+    expect(evaluate('encodeURI("Hello World")')).to.equal('Hello%20World');
+    expect(evaluate('encodeURIComponent("#foo=bar")')).to.equal('%23foo%3Dbar');
```
Comment (choumx) [inline extensions/amp-bind/0.1/test/test-bind-expression.js]: Is this a fragment or URL params? We should test the latter.

Comment (kmh287) [inline extensions/amp-bind/0.1/test/test-bind-expression.js]: done

File: extensions/amp-bind/0.1/test/test-bind-expression.js [hunk 2]
```diff
@@ -268,6 +268,13 @@ describe('BindExpression', () => {
     }).to.throw(unsupportedFunctionError);
   });
 
+  it('should support encodeURI and encodeURIComponent', () => {
+    expect(evaluate('encodeURI("http://www.google.com/s p a c e.html")'))
+        .to.equal('http://www.google.com/s%20p%20a%20c%20e.html');
+    expect(evaluate('encodeURIComponent("hello world")'))
+        .to.equal('hello%20world');
```
Comment (choumx) [inline extensions/amp-bind/0.1/test/test-bind-expression.js]: Nit: This test for `encodeURIComponent` tests the same functionality as the one for `encodeURI`. Recommend testing proper encoding for some of these chars: `; , / ? : @ & = + $ #`

Comment (kmh287) [inline extensions/amp-bind/0.1/test/test-bind-expression.js]: I'm not sure that's necessary. We don't need to test the correctness of the functions, just that using them in bind expressions works.

Comment (choumx) [inline extensions/amp-bind/0.1/test/test-bind-expression.js]: Correctness is currently an implementation detail. 😄  It is a nitpick though.

**Pull request merged**